app/views/projects/_form.html.erb

```diff
@@ -8,9 +8,9 @@
 <div>
   <span id="project_entry_form"></span>
   <div class="hidden-print">
-    <button id="toggle-expand-all-panels" class="btn btn-info btn-xs" title="Expand all panels"type="button">Expand panels</button>
-    <button id="toggle-show-all-details" class="btn btn-info btn-xs" title="Show all details" type="button">Show all details</button>
-    <button id="toggle-hide-metna-criteria" class="btn btn-info btn-xs" title="Hide met &amp; N/A criteria (leaving unmet and unknown)" type="button">Hide met &amp; N/A</button>
+      <button id="toggle-expand-all-panels" class="btn btn-primary btn-xs btn-stack" title="Expand all panels"type="button">Expand panels</button>
+      <button id="toggle-show-all-details" class="btn btn-primary btn-xs btn-stack" title="Show all details" type="button">Show all details</button>
+      <button id="toggle-hide-metna-criteria" class="btn btn-primary btn-xs btn-stack" title="Hide met &amp; N/A criteria (leaving unmet and unknown)" type="button">Hide met &amp; N/A</button>
```

Comment:
Can you change these back to btn-info,  we chose that style for the color scheme.